MainWindow.py

```
@@ -1,695 +1,716 @@
+				saveInfo.State #谱面状态
+				))
+		# print(songNameJson["NotDLCSong"])
+		if not self.dataSortMethodsort[0] is None:
+			self.SortClick(self.dataSortMethodsort)
+		self.InitLabel('数据展示生成完成.',close=True)
+		endTime = time.perf_counter_ns()
+		self.logger.debug("DataLoad Run Time: %f ms"%((endTime - startTime)/1000000))
+		self.UpdateWindowInfo()
+
+# 控件更新功能组
+	def ShowMessageBox(self, type:int=0, title:str="", msg:str=""):
```

Ruff (0.14.8)

Unused method argument: `type`

(ARG002)